contrib/setup

```diff
@@ -32,6 +44,9 @@ setup_deps()
 
 setup_run_dev()
 {
+    if [ "$SELINUX" -eq "1" ]; then
+        return
```

Comment:
I would suggest still emitting a message here to let people know /usr/local isn't going to fly BECAUSE of selinux though